README.md

```diff
@@ -39,45 +31,24 @@ In addition to the steps above there are a few addition steps required for trans
 
 #### New Language
 
-1. Create a folder using the 2 character code (e.g. ja, en, es, etc) with lesson subfolders:
+1. Create a folder using the ISO language code (e.g. ja, zh-hans, es, et al) with lesson subfolders:
 
   ```shell
   $ cd elixirschool
   $ mkdir -p ja/lessons/{basics,advanced,specifics,libraries}
-  $ touch ja/lessons/{basics,advanced,specifics,libraries}/.gitignore
+  $ touch ja/lessons/{basics,advanced,specifics,libraries}/.gitkeep
   ```
 
-1. Update `_config.yml` by including the 2 character code in `languages` and adding translations to `sections`, `description` and `toc`:
+1. Create a locale file for your new language using `_data/locales/en.yml` as a guide:
 
-  ```yaml
-  languages: ['en', 'ja']
-  default_lang: en
-  exclude_from_localization: []
-  sections:
-    - tag: basics
-      label:
-        en: Basics
-        ja: 基本
-
-  description:
-    en: Lessons about the Elixir programming language
-    ja: プログラミング言語Elixirのレッスン
-
-  toc:
-    en: Table of Contents
-    ja: 目次
+  ```shell
+  $ touch _data/locales/ja.yml
   ```
 
-1. If the new language is RTL (right-to-left) it should also be added to the `rtl_languages` list:
+1. If the new language is RTL (right-to-left) it should be added to the `rtl_languages` list in `config.yml`:
 
   ```yaml
-  rtl_languages: ['ar']
-  ```
-
-1. Add it to list in `index.md`:
-
-  ```markdown
-  Available in [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl] [Português][pt], [Русском][ru] and [Bahasa Melayu][ms] and other.
+  script_direction: rtl
   ```
 
 #### Translated Lesson
```

Comment:
we also need to add that they need to add their language to `_data/interlang.yml` , create a new list, and also add the language item to every list there

Reply:
If it's a new lesson, add lesson to `_data/contents.yml`

Reply:
There's so many more steps now!  😓